static/js/ducks/sources.js

```diff
@@ -14,8 +18,15 @@ export function fetchSources(filterParams = {}) {
   return API.GET(`/api/sources?${queryString}`, FETCH_SOURCES);
 }
 
+export function fetchGroupSources(filterParams = {}) {
+  const params = new URLSearchParams(filterParams);
```

Comment:
Check whether group_ids is present as expected

Reply:
What's the right way to raise an exception if the group_id parameter is not there?

Reply:
this is actually probably fine to leave as-is